org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/apps/form/WPayPrint.java

```diff
@@ -660,31 +575,18 @@ public void onCallback(Boolean result)
 	 */
 	protected boolean getChecks(String PaymentRule)
 	{
-		//  do we have values
-		if (m_C_PaySelection_ID <= 0 || m_C_BankAccount_ID == -1
-			|| fPaymentRule.getSelectedIndex() == -1 || fDocumentNo.getValue() == null)
-		{
-			Dialog.error(m_WindowNo, "VPayPrintNoRecords",
-				"(" + Msg.translate(Env.getCtx(), "C_PaySelectionLine_ID") + "=0)");
-			return false;
-		}
-
-		if (log.isLoggable(Level.CONFIG)) log.config("C_PaySelection_ID=" + m_C_PaySelection_ID + ", PaymentRule=" +  PaymentRule);
+		AtomicReference<ValueNamePair> error = new AtomicReference<>();
 		
-		//	get payment selection checks without check no assignment
-		m_checks = MPaySelectionCheck.get(m_C_PaySelection_ID, PaymentRule, null);
-
-		//
-		if (m_checks == null || m_checks.length == 0)
+		boolean ok = getChecks(PaymentRule, fDocumentNo.getValue(), error, PaymentRule);

```

Comment:
Hi @hengsin - found a problem in this line with the last parameter that is trxName.

Can you please add this patch:
[pr1554_PR.txt](https://github.com/idempiere/idempiere/files/9961685/pr1554_PR.txt)

it fixes too two minor additional issues

Reply:
done.